src/eventHandlers/workerInitRequest.ts

```diff
@@ -47,6 +47,9 @@ export function workerInitRequest(channel: WorkerChannel, requestId: string, _ms
     }
 
     logColdStartWarning(channel);
+    if (_msg.functionAppDirectory) {

```

Comment:
nit: Rename to `msg`. We only use `_` to prefix unused variables

Reply:
Done